From c1bd0ce30aa7e94748de4a39aceeaa1b8c98d72a Mon Sep 17 00:00:00 2001
From: Zahari Doychev <zdoychev@maxlinear.com>
Date: Thu, 22 Oct 2020 15:25:35 +0200
Subject: [PATCH] tc_colmark: export colmark flags

Function for getting the colmark flags is added to the action API.
---
 include/net/tc_act/tc_colmark.h | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/include/net/tc_act/tc_colmark.h b/include/net/tc_act/tc_colmark.h
index 2b5870d70b4b..db555160c876 100644
--- a/include/net/tc_act/tc_colmark.h
+++ b/include/net/tc_act/tc_colmark.h
@@ -39,4 +39,9 @@ static inline u8 tcf_colmark_mtype(const struct tc_action *a)
 	return to_colmark(a)->type;
 }
 
+static inline u8 tcf_colmark_flags(const struct tc_action *a)
+{
+	return to_colmark(a)->flags;
+}
+
 #endif /* __NET_TC_COLMARK_H */
